src/nrc/accounts/apps.py

```diff
@@ -1,5 +1,29 @@
-from django.apps import AppConfig
+from io import StringIO
+
+from django.apps import AppConfig, apps
+from django.contrib.contenttypes.management import create_contenttypes
+from django.core.management import call_command
+from django.db.models.signals import post_migrate
+
+
+def update_admin_index(sender, **kwargs):
```

Comment:
Is this to delete the old AppGroups from django-auth-adfs-db?

Reply:
yes and no - we had a datamigration loading the fixture `default_admin_index.json`, but that only happens once. In new versions of ON, that fixture needs to be loaded manually. By hooking into the post-migrate signal, the fixture is reloaded on every deploy, so we can control the menu structure properly from the code.